### ISSUE TITLE
ImportError: No module named dbus

### ISSUE THREAD

**Issue by anselal** [dushyantRathore/Crypto-Notifier#1]
I installed the packages from requirements.txt but still get this error message. What python version are you using and in which OS have you tested the app ??

**Comment by dushyantRathore** [comment]
I am using python 2.7.12 and I have tested the code on Linux.

**Comment by anselal** [comment]
I see. In windows it is a bit tricky to install dbus-python and that's why I could not run the app.
And because of that, I couldn't didn't #2 and #3. Can you confirm that those two pull requests actually work ???

**Comment by dushyantRathore** [comment]
Yes, I ran the code after the merged pull requests and it is working fine. Will check for dbus-python and alternate options in windows and let you know.

You could refer to https://github.com/jithurjacob/Windows-10-Toast-Notifications for windows 10 implementation
